pkg/workloads/console/events/publisher.go

```diff
@@ -0,0 +1,24 @@
+package events
+
+import "context"
+
+type Healther interface {
+	Health(context.Context) error
+}
+
+type Publisher interface {
+	Healther
+	Publish(context.Context, interface{}) (string, error)
+}
+
+var _ Publisher = &NopPublisher{}
+
+// https://en.wikipedia.org/wiki/NOP_(code)
+type NopPublisher struct{}
```

Comment:
Interesting - I'd usually tend towards `Noop` in this kind of context, but this also feels right

Reply:
I did consider a few names here but opted to more closely match the instruction name (for no particular reason).

Reply:
TIL